docs/Guides/Scim/Patch/SCIM-Patch-supported-operations.md

```diff
@@ -0,0 +1,124 @@
+---
+stoplight-id: fdxg63veh82v5
```

Comment:
Remove stoplight-id

Reply:
Actually the stoplight id should be kept. It is used to provide stable URL paths for the public website in the event that we move where the file is stored within the repo. 
https://meta.stoplight.io/docs/platform/hosted-docs/stoplight-urls#about-stable-ids